### DISCUSSION TITLE
Another successful build, thank you!

### DISCUSSION THREAD

**Discussion by mockendon** [kachurovskiy/nanoels#32] [General]
On my Chester DB7 (Rebadged MiniLathe).
![IMG_3805](https://user-images.githubusercontent.com/12686546/146538525-102b3795-918e-4d29-a6b7-d2073add2419.jpeg)
![IMG_3804](https://user-images.githubusercontent.com/12686546/146538532-636912b0-1a40-434e-82f8-f533f50cae60.jpeg)
![IMG_3806](https://user-images.githubusercontent.com/12686546/146538567-957fd34b-83a3-4eee-8784-074d853db60e.jpeg)

Jogging in 1mm steps

https://user-images.githubusercontent.com/12686546/146538813-ba5fe33f-333b-45cd-b657-85618b4a2c32.mov

**Comment by kachurovskiy** [comment]
Looks great! Glad it worked. Congratulations on making it through.
